nextcord/auto_moderation.py

```diff
@@ -77,6 +77,22 @@ class AutoModerationTriggerMetadata:
 
             This is ``None`` and cannot be provided if the trigger type of the rule is not
             :attr:`AutoModerationTriggerType.keyword`.
+    regex_patterns: Optional[List[:class:`str`]]
+        A list of regex patterns which will be matched with content.
+
+        .. versionadded:: 2.4
+
+        .. note::
+
+            This is ``None`` and cannot be provided if the trigger type of the rule is not
+            :attr:`AutoModerationTriggerType.keyword`.
+
+        .. warning::
+
+            The flavor of regex used is Rust flavor and has no guarantees of working with
+            the ``re`` module.
```

Comment:
```suggestion
            the :mod:`re` module.
```
<https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#role-py-mod>